data/human/paradise world jobs.txt

```diff
@@ -500,6 +502,58 @@ mission "Paradise Job: Retirees"
 		dialog "You bid goodbye to the retirees, who were somewhat prone to spilling drinks or irritating you whilst attempting to do work. They were otherwise extremely polite and full of highly amusing, though often morally questionable, anecdotes. Once the last of their belongings have been safely unloaded your accounts receive the payment of <payment> from their travel broker."
 
 
+mission "Paradise Job: Pirate Retirees (Illegal)"
+	name "Wealthy retirees to <planet>"
+	job
+	repeat
+	description `This coterie of <bunks> wealthy "retirees" is willing to pay <payment> for stylish, no-questions-asked transport to <destination>, along with <tons> of their most treasured possessions.`
+	passengers 4 10 .6
+	cargo "treasured possessions" 4 10 .2
+	illegal 15000 `In addition to the fine, the "retirees" and their possessions are removed from your ship.`
```

Comment:
This message is far too short. Why are they removed from my ship? The job description does imply that there's something suspect going on, but then I learn nothing when scanned? Look to the wanted passenger pirate jobs for the type of messages we should have for illegal passengers.

Reply:
All right, I've changed that, and it should be better now. I was going off of the drug running missions before, that was obviously a mistake.